packages/jsii-pacmak/test/expected.jsii-calc/java/src/main/java/software/amazon/jsii/tests/calculator/RootStruct.java

```diff
@@ -0,0 +1,156 @@
+package software.amazon.jsii.tests.calculator;
+
+/**
+ * This is here to check that we can pass a nested struct into a kwargs by specifying it as an in-line dictionary.
+ * 
+ * This is cheating with the declared types, but Python people don't play by
+ * the rules much apparently.
```

Comment:
I know y'all are opinionated but having pages and pages of this comment could be considered offensive by those using the target language as they are obviously playing by the rules.

Reply:
You are absolutely right. This is not appropriate. We’ll remove this comment.

Reply:
Apologies - this was meant in a playful tone but didn't come across as intended & is indeed inappropriate.

Edited version at #975

Reply:
Yeah maybe "Python doesn't play by the same rules." Appreciate y'alls work.